src/main/java/com/iexec/worker/feign/CustomFeignClient.java

```diff
@@ -107,19 +107,21 @@ public void registerWorker(WorkerConfigurationModel model) {
     }
 
     public List<InterruptedReplicateModel> getInterruptedReplicates(long lastAvailableBlockNumber) {
+        List<InterruptedReplicateModel> interruptedReplicates = null;
```

Comment:
The list should be initialized with new ArrayList(), not with a null. Then we can just return the list (empty or not)

Reply:
Oh :o !